es6-operators/main.js

```diff
@@ -0,0 +1,91 @@
+/* eslint-disable no-unused-vars */
+const v1 = { value: 'Something' };
+const v2 = '';
+const v3 = undefined;
+const v4 = [3, 5, 7, 9];
+
+/* TODO:
+ * Demonstrate how to execute an expression only if a variable is truthy.
```

Comment:
So,... you did it in one expression, which is good, but you ended up using assignment instead of expression execution. So you actually log twice. You are supposed to log only once. Try again.